server/uploadHandler/src/createNewJob.ts

```diff
@@ -53,8 +65,8 @@ export const createNewJob = async (eventBody: string): Promise<FileUploadRespons
 
 const sqsTriggerModels = async (jobID: string) => {
 	const uploadedFiles = await getUploadedFilesInJob(jobID);
-	for (let i = 0; i < uploadedFiles.length; i++) {
-		const fileKey = uploadedFiles[i];
+	for (const i=0;i < uploadedFiles.length; i++) {
```

Comment:
Errr, don't think this will work? Leaving it as let is fine
![image](https://user-images.githubusercontent.com/41775764/133396353-852528e8-0735-4ed6-b2fc-4393f32603e8.png)